lib/src/provider.dart

```diff
@@ -48,7 +48,11 @@ class OidcProvider extends OAuthProvider {
           ),
         );
       } else {
-        credential = await auth.signInWithProvider(firebaseAuthProvider);
+        if (kIsWeb) {
```

Comment:
Please un-nest this into an `else if`/`else` on the enclosing `if` statement

Reply:
Done